ui/src/app/applications/components/application-details/application-resource-list.tsx

```diff
@@ -24,12 +24,13 @@ export const ApplicationResourceList = ({
                 <div className='columns small-2 xxxlarge-2'>NAME</div>
                 <div className='columns small-2 xxxlarge-2'>GROUP/KIND</div>
                 <div className='columns small-1 xxxlarge-2'>SYNC ORDER</div>
-                <div className='columns small-3 xxxlarge-3'>NAMESPACE</div>
-                <div className='columns small-2 xxxlarge-2'>STATUS</div>
+                <div className='columns small-2 xxxlarge-2'>NAMESPACE</div>
+                <div className='columns small-2 xxxlarge-2'>CREATED AT</div>
+                <div className='columns small-1 xxxlarge-1'>STATUS</div>
             </div>
         </div>
         {resources
-            .sort((first, second) => nodeKey(first).localeCompare(nodeKey(second)))
+            .sort((first, second) => -createdOrNodeKey(first).localeCompare(createdOrNodeKey(second)))
```

Comment:
is the negative here intentional?

Reply:
yes, I would like the most recent ones to be displayed at the top